mocked-api/schemasData.ts

```diff
@@ -39,6 +39,14 @@ export const schemaCatalogData = [
     type: "action",
     href: "/api/v1/schemas/actions/aws_lambda_sink_0.1",
   },
+  {
+    kind: "ProcessorSchemaEntry",
+    id: "google_pubsub_sink_0.1",
+    name: "Google PubSub",
+    description: "Send the event to Google PubSub",
+    type: "action",
+    href: "/api/v1/schemas/actions/google_pubsub_sink_0.1",
```

Comment:
please update the href to be aligned with latest main

Reply:
Done.